app.rb

```diff
@@ -42,7 +54,16 @@ def list_all_labels
   end
 
   def list_all_authors
-    puts 'List of authors'
+    puts 'List of authors:'
+    @authors.each do |author|
+      puts "#{author.firstname} #{author.lastname} (ID: #{author.id})"
+      author.items.each do |item|
+        if item.is_a?(Game)
+          puts " - Game: #{item.title} (Players: #{item.multiplayers}), Last Played: #{item.lastplayed_date}"
+          # Add similar logic for other item types (books, music albums, etc.)
```

Comment:
I really liked how you let the user know what is the type of item you are creating.